pkg/service/redisstore.go

```diff
@@ -227,17 +256,58 @@ func (s *RedisStore) DeleteParticipant(_ context.Context, roomName livekit.RoomN
 	return s.rc.HDel(s.ctx, key, string(identity)).Err()
 }
 
+// Temporary migration for upgrading from LiveKit v1.1.2 to v1.1.3
+func (s *RedisStore) MigrateEgressInfo() (int, error) {
```

Comment:
do we have any sort of marker (maybe in redis) indicating the last version of livekit? might be useful for future "migration" efforts.

Reply:
also. wondering how this works when multiple instances of lk server are running at the same time.

Reply:
added version and migration locking